static/scripts/topicEdit.js

```diff
@@ -1,4 +1,6 @@
-const {SortableContainer, SortableElement, SortableHandle, arrayMove} = window.SortableHOC;
+import React from 'react';
```

Comment:
Nur für mich zum verständnis: Wieso wird hier React importiert, wenn man sowieso die vendor-react.js einbinden muss? Wird dann nicht React doppelt geladen?
Ansonsten gefällts mir 👍

Reply:
Nein, React wird nicht doppelt importiert und es ist sogar zwingend notwendig React an der Stelle zu importieren. Andernfalls wird er einen Fehler werfen, dass React nicht gefunden werden kann. Auch wenn eine `vendor-react.js` von Webpack gebaut wird, ist React nämlich nicht direkt global zugänglich. 
Tatsächlich ist das Webpack's Werk. Wenn wir uns das window object und die minified `vendor-react.js` schauen, sehen wir auch was passiert:
Hier sehen wir, dass webpack die minified version von react und react-dom in ein Array im globalen window pushed:
![image](https://user-images.githubusercontent.com/7314829/42581903-1c37ad3e-852e-11e8-9ef2-49bee8a57ec7.png)
So sieht das dann aus (hinter diesen einzelnen Einträgen verstecken sich react und react-dom):
![image](https://user-images.githubusercontent.com/7314829/42581949-38ee176a-852e-11e8-8e72-7dd67c5727db.png)
Und wenn wir jetzt in das minified `topicEdit.js` gucken würden, sähen wir, dass webpack dort automatisiert den `import React from 'react'` mit genau diesem dynamischen `window.webpackJsonp` object verheiratet hat. :)

Reply:
Ahhh, endlich habe ich das mal verstanden. Dankeschön :)